### ISSUE TITLE
populate all .shed.yaml with descriptions and synopses from MTS

### ISSUE THREAD

**Issue by martenson** [galaxyproject/tools-devteam#246]

**Comment by erasche** [comment]
https://gist.github.com/erasche/4ac3448b036f09979e14 there's a reason I named the script what I did, it's pretty ugly, sorry.

**Comment by martenson** [comment]
@erasche did you run it on IUC repo?

**Comment by erasche** [comment]
@martenson yep, that's what I used for the IUC repo a long time ago. @peterjc used that as well for his repositories but I imagine he cleaned it up a bit

**Comment by peterjc** [comment]
I did use something based on it, yes. It got the job done and was a useful learning exercise for working with BioBlend. I probably still have a copy of my edited version on my work machine... but it was still very rough.

**Comment by martenson** [comment]
Touched again in https://github.com/galaxyproject/planemo/pull/325
@peterjc could you please check if you can find the script you used? Thanks.

**Comment by peterjc** [comment]
See https://gist.github.com/peterjc/5ebbf446d799f3aaa639 for my ``pulldown_shed_yml.py`` script for populating Galaxy ``.shed.yml`` files with metadata already in Tool Shed

**Comment by peterjc** [comment]
See also https://gist.github.com/peterjc/2cf9fa7e2dc921d27fcb which was even less documented but also written around the same time (file was last modified 29 April 2015).
